src/main.cpp

```diff
@@ -235,31 +257,51 @@ class WirelessFTNode : public rclcpp::Node
     return result;
   }
 
+  // Check if alive, and if not, re-init
+  void keep_alive_callback() {
+    if (!mIsAlive) {
+      RCLCPP_WARN(get_logger(), "KeepAlive Failed, re-init");
+      // Disconnect and Re-connect
+      mWFT->udpClose();
+      mWFT->telnetDisconnect();
+
+      this->init();
+      return;
+    }
+    mIsAlive = false;
+  }
+
   // For reading from F/T Sensor and publishing wrench
-  void timer_callback()
-  {
+  void timer_callback() {
     auto packet = mWFT->readDataPacket();
     if (!packet.valid) {
-      RCLCPP_WARN(get_logger(), "Skipping Invalid Packet");
+      //RCLCPP_WARN(get_logger(), "Skipping Invalid Packet");
```

Comment:
Can you keep this in, at least as a debug log? I've found it valuable to indicate that there may be an error in the FT sensor.